core/domain/user_query_domain.py

```diff
@@ -21,22 +21,17 @@
 
 import collections
 
+from constants import constants
 import feconf
 import python_utils
 import utils
 
+__attribute_names__ = [
+    predicate['backend_attr'] for predicate in (
+        constants.EMAIL_DASHBOARD_PREDICATE_DEFINITION)]
```

Comment:
Why is the `attribute_names` enclosed by `__`?

Reply:
Using attribute_names failed the lint check so I used enclosed it with __ because it is only used locally.

Reply:
What is the exact lint error?